dts/bindings/spi/nxp,s32-spi.yaml

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP S32 SPI controller
+
+compatible: "nxp,s32-spi"
+
+include: [spi-controller.yaml, pinctrl-device.yaml]
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  num-cs:
+    type: int
+    required: true
+    description: |
+      The number of the Chip Select signals.
+
+  clock-frequency:
+    type: int
+    required: true
+    description: |
+      Module clock frequency in Hz.
+
+  pinctrl-0:
+    required: true
+
+  pinctrl-names:
+    required: true
+
+  slave:
```

Comment:
Which of these properties are needed. Does'nt look like all are being used.

Reply:
they are all used, e.g. https://github.com/zephyrproject-rtos/zephyr/blob/c80232f16d477906346cab12b60a17794148bcbc/drivers/spi/spi_nxp_s32.c#L658-L660

Reply:
I don't see any values being set for these properties in the dtsi files. Can you please check.